doc/DECRYPTION_EXAMPLE.md

```diff
@@ -0,0 +1,123 @@
+
+Depending on the size of the file, the job may take some time.  If the job is not yet complete, this will give us status information.  Shown below are the results for our completed job.  We can download the file from the URL provided.
+
+Take special note of the new `KeyMap` section of the response.  To decrypt the file, we will need the filename (the first part of the keymap) and the [symmetric key](../ENCRYPTION.md#how-we-encrypt) (the second part of the keymap), as shown above.  There are no spaces in either one.
+
+- Copy these values (filename and symmetric key) for later.
+
+Our last API task is to download the encrypted file. 
+
+- Open the data file section in Swagger (click `/api/v1/jobs/{jobID}/{filename}`)
+- Paste the job ID and filename into the appropriate boxes
+- Click `Execute`
+
+<img src="images/decrypt_demo_08.png" alt="Get data file (Swagger)" width="600" />
+
+- Click the `Download file` link that appeared in the response section.  Note that a large sample file may take a while to download.
+
+<img src="images/decrypt_demo_09.png" alt="Download file" width="600" />
+
+After downloading the file, we'll move to the command line.  Navigate to the directory you saved `decrypt.py` and `requirements.txt` in [earlier](#gathering-the-tools).
+
+<img src="images/decrypt_demo_10.png" alt="Directory with decryption tool" width="450" />
+
+Let's verify that Python is running properly.  
+- Run `decrypt.py` with the help argument (`python decrypt.py -h`).  You should get the response shown below.
+
+<img src="images/decrypt_demo_11.png" alt="Decrypt.py syntax" width="600" />
+
+- Rename the downloaded file with the filename you saved earlier.
+
+<img src="images/decrypt_demo_12.png" alt="Rename downloaded file" width="600" />
+
+We're now ready to decrypt the file!  Our sample decryption tool will print the decrypted contents to the console, so we'll send the output to a file.  Make sure to use the following syntax, with the entire command on the same line:
+
+    python decrypt.py 
+        --pk   [location_of_private_key] 
+        --file [location_of_encrypted_file] 
+        --key  [symmetric_key_value]
+
+<img src="images/decrypt_demo_13.png" alt="Running decrypt.py" width="900" />
+
+Take a look at the result.  If you do not see unencrypted [NDJSON](http://ndjson.org/) (two example lines shown below), then skip ahead to the [troubleshooting](#troubleshooting) section.
+
+<img src="images/decrypt_demo_14.png" alt="Parsing decrypted file contents" width="900" />
+
+### Reading further
+* [Encryption documentation](../ENCRYPTION.md)
+* [API documentation](../API.md)
+
+### Troubleshooting
+#### Authentication problems in Swagger
+- Has your token expired?  [Ask the support team](https://groups.google.com/forum/#!forum/bc-api) for a new token.
+- Is the lock on the `Authorize` icon not closed?  Click it again, and after pasting your token in the `api_key` box, make sure to click the `Authorize` button.
```

Comment:
I have clicked the logout button a few times by mistake, might be worth noting that.